class BaseError(Exception):
    pass


class UserError(BaseError):
    pass
